url.bs

```diff
@@ -3,7 +3,7 @@ Group: WHATWG
 H1: URL
 Shortname: url
 Text Macro: TWITTER urlstandard
-Abstract: The URL Standard defines URLs, domains, IP addresses, the <code title>application/x-www-form-urlencoded</code> format, and their API.
+Abstract: The URL Standard defines URLs, domains, IP addresses, the <code><a>application/x-www-form-urlencoded</a></code> format, and their API.
```

Comment:
I don't think we want links in abstracts.

Reply:
Why not? I assume that's what the `title` was supposed to be about.

Reply:
The empty title was there to prevent a link.

Reply:
The idea is that abstracts stand on their own. If you need to click on links it has kinda failed. Perhaps that means this abstract is not super great though.

Reply:
Happy to remove it, but I think links in abstracts are nice, just like they are anywhere else.